docs/developer-guide.md

```diff
@@ -1,5 +1,9 @@
 # Developer Guide
 
+## Table of Contents
+- [How to Debug](#how-to-debug)
```

Comment:
We'd have to provide both local and remote debugging (we can start with local but we should distinguish).